### PR TITLE
new package: Mendeley

### PR DESCRIPTION
**Pull request by badmutex**
This request is to add the Mendeley Desktop reference manager for 32 and 64 bit Linux.

### CODE REVIEW

File: pkgs/applications/office/mendeley/mendeley.nix [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    librarypath="${stdenv.lib.makeLibraryPath deps}:$out/lib:$out/lib/qt"
+    wrapProgram $out/bin/mendeleydesktop --prefix LD_LIBRARY_PATH : "$librarypath"
+  '';
+
+  dontStrip = true;
+  dontPatchElf = true;
+
+  meta = {
+    homepage = http://www.mendeley.com;
+    description = "Mendeley is a free reference manager and academic social network.";
```
Comment (pSub) [inline pkgs/applications/office/mendeley/mendeley.nix]: `description` should not contain the package name and end with dot. See http://nixos.org/nixpkgs/manual/#idm140737319320832

**Comment by lethalman** [comment]
Built locally, thanks.

**Comment by bjornfor** [comment]
```
$ nix-env -f . -qa '*' --meta --xml --drv-path --show-trace
error: while querying the derivation named ‘mendeley-1.13.1-stable_amd64’:
while evaluating the attribute ‘installPhase’ of the derivation ‘mendeley-1.13.1-stable_amd64’ at "/home/bfo/nixpkgs/pkgs/applications/office/mendeley/default.nix":34:3:
while evaluating ‘makeSearchPath’ at "/home/bfo/nixpkgs/lib/strings.nix":44:28, called from "/home/bfo/nixpkgs/pkgs/applications/office/mendeley/default.nix":52:20:
while evaluating ‘concatStringsSep’ at "/home/bfo/nixpkgs/lib/strings.nix":34:33, called from "/home/bfo/nixpkgs/lib/strings.nix":45:5:
while evaluating ‘fold’ at "/home/bfo/nixpkgs/lib/lists.nix":20:19, called from "/home/bfo/nixpkgs/lib/strings.nix":35:5:
while evaluating ‘fold'’ at "/home/bfo/nixpkgs/lib/lists.nix":23:15, called from "/home/bfo/nixpkgs/lib/lists.nix":27:8:
while evaluating anonymous function at "/home/bfo/nixpkgs/lib/strings.nix":15:32, called from "/home/bfo/nixpkgs/lib/lists.nix":26:14:
while evaluating anonymous function at "/home/bfo/nixpkgs/lib/strings.nix":45:32, called from undefined position:
attribute ‘gcc’ missing, at "/home/bfo/nixpkgs/pkgs/applications/office/mendeley/default.nix":25:5
```

gcc -> cc?

**Comment by bjornfor** [comment]
Fixed in master.

**Pull request merged**